python-package/hydro_serving_grpc/timemachine/reqstore_client.py

```diff
@@ -7,8 +7,22 @@
 
 from hydro_serving_grpc.timemachine.reqstore_service_pb2_grpc import *
 from hydro_serving_grpc.timemachine.reqstore_service_pb2 import *
+from hydro_serving_grpc.monitoring.metadata_pb2 import ExecutionError
+import requests
```

Comment:
We could also move this import statement into the class ctor and handle import error with detailed description that `requests` needs to installed separately.